FGEGraphics/UISystem/UIElement.cs

```diff
@@ -683,6 +670,11 @@ public virtual IEnumerable<UIElement> GetChildrenNotAt(int x, int y)
         }
     }
 
+    /// <summary>Returns whether this element (and recursive children) can be interacted with at the specified position.</summary>
```

Comment:
`recursive parents` rather than `children`

Reply:
The parent elements constrain the children in this case even though the call is propagated up the hierarchy

Reply:
Ah, I see what you mean - possibly there's a way to phrase this to make the idea more clear? I think what threw me was the emphasized word `recursive` implying that there is recursive logic applied here (and there is just unrelated). I think the `constrain` keyword is an important one - so maybe `Returns whether this element can be interacted with at the specific location. Note that this also constrains whether or not the element's children can be interacted with at that location as well.`